tools/istio-docker.mk

```diff
@@ -234,7 +235,12 @@ dockerx:
 dockerx.%:
 	@DOCKER_TARGETS=docker.$* BUILD_ALL=false $(MAKE) --no-print-directory -f Makefile.core.mk dockerx
 
-docker.base: docker/Dockerfile.base
+docker.base: docker.ubuntu docker.distroless
```

Comment:
I'm not sure about switching from base to ubuntu and what repercussions that has in the short term.

I know you would need to update all the Dockerfile.* files that use istio/base. I know I just hardcoded it in some release-builder code as well.

Reply:
I'm not sure if anyone outside of Istio would care about the base vs ubuntu within their pipelines, but not sure.

Reply:
I reverted the rename of ubuntu to base. While the name isn't accurate, for the moment this is the only base image available that is not distroless.

Cheers,
-steve